src/components/QuranReader/ReadingPreferenceSwitcher.module.scss

```diff
@@ -0,0 +1,4 @@
+.container {
+  max-width: calc(20 * var(--spacing-large));
```

Comment:
1. this should be a const
2. we should do a css min between this value and 90%. because we don't want it to occupy the full width on small devices.

![image](https://user-images.githubusercontent.com/11590314/136687927-a90e760c-286f-464f-a9f6-fcc237748604.png)